### ISSUE TITLE
torch version

### ISSUE THREAD

**Issue by linhanxiao** [yufengm/SelfAttentive#2]
thank you for  Implementation of A Structured Self-attentive Sentence Embedding,Do you have the torch
implementation version

**Comment by yufengm** [comment]
I'm sorry that I just implement it in pytorch.

**Comment by linhanxiao** [comment]
Can I ask you about the torch implementation  version, in your pytorch implementation,the penal  was directly added to entropyloss and loss.backward(),in torch implementation,what should i do to implement
the similar function,should icreate a new class inherited from nn.Criterion to implement  it?
